cmd/state-installer/cmd.go

```diff
@@ -296,6 +298,17 @@ func execute(out output.Outputer, cfg *config.Instance, an analytics.Dispatcher,
 			return errs.Wrap(err, "Could not check if install path is empty")
 		}
 		if !empty {
+			if usingDefaultInstallPath {
+				// We're having trouble pinning down why these errors are occurring, so report the list of
+				// existing files to Rollbar to help diagnose.
+				if files, err := os.ReadDir(params.path); err == nil {
+					fileList := []string{}
+					for _, file := range files {
+						fileList = append(fileList, filepath.Join(params.path, file.Name()))
+					}
+					rollbar.Critical("Installation path must be an empty directory: %s\nExisting files:\n%s", params.path, strings.Join(fileList, "\n"))
```

Comment:
```suggestion
				if files, err := fileutils.ListDirSimple(params.path, false); err == nil {
					rollbar.Critical("Installation path must be an empty directory: %s\nExisting files:\n%s", params.path, strings.Join(files, "\n"))
```

Reply:
`fileutils.ListDirSimple()` does a recursive listing, which this ticket explicitly said not to do. Should I still use your suggestion?

Reply:
Ahh my bad, no; thanks for spotting that!